src/winetricks

```diff
@@ -10174,25 +10174,485 @@ _EOF_
+
+				# mf.dll
+				## Grab from win7sp1
+				helper_win7sp1_export amd64_microsoft-windows-mediafoundation_31bf3856ad364e35_6.1.7601.17514_none_fa8534ab236134c4 mf.dll 64
+				## z0z0z mirror
+				w_download_wrapper https://github.com/z0z0z/mf-install/raw/master/system32/mf.dll mf.dll dbf0458bed308078edd82c81e6d1209e111c5bfa5d64dd068cedfd9e0a6c05ab 64
+
+				# mfreadwrite.dll
+				helper_win7sp1_export amd64_microsoft-windows-mfreadwrite_31bf3856ad364e35_6.1.7601.17514_none_177bed732ea3f85f mfreadwrite.dll 64
+				## z0z0z mirror
+				w_download_wrapper https://github.com/z0z0z/mf-install/raw/master/system32/mfreadwrite.dll mfreadwrite.dll 665d8a6cdf29f9df2d9014a0eb7dcf1f840b7809a625f0752fdc3830ee952413 64
 
-    if [ "$W_ARCH" = "win64" ]; then
-        helper_win7sp1_x64 amd64_microsoft-windows-mediafoundation_31bf3856ad364e35_6.1.7601.17514_none_fa8534ab236134c4/mf.dll
-        w_try cp "$W_TMP/amd64_microsoft-windows-mediafoundation_31bf3856ad364e35_6.1.7601.17514_none_fa8534ab236134c4/mf.dll" "$W_SYSTEM64_DLLS/mf.dll"
-    fi
+				# sqmapi.dll
+				helper_win7sp1_export amd64_microsoft-windows-sqmapi_31bf3856ad364e35_6.1.7601.17514_none_5c63b87c1b6dc7ec sqmapi.dll 64
+				## z0z0z mirror
+				w_download_wrapper https://github.com/z0z0z/mf-install/raw/master/system32/sqmapi.dll sqmapi.dll 17411c6a6c1e699bc4b0c04d782fd9aa09cf577dba41e743f7588904d489cb9f 64
+
+				# wmadmod.dll
+				helper_win7sp1_export amd64_microsoft-windows-wmadmod_31bf3856ad364e35_6.1.7601.17514_none_e474f0ac34979bc4 wmadmod.dll 64
+
+				# wmvdecod.dll
+				helper_win7sp1_export amd64_microsoft-windows-wmvdecod_31bf3856ad364e35_6.1.7601.17514_none_20b089c0f6efacae wmvdecod.dll 64
+		fi
+
+		# Generate mf.reg
+		cat > "$W_CACHE/$W_PACKAGE/mf.reg" <<_EOF_
+		REGEDIT4
+
+		[HKEY_LOCAL_MACHINE\Software\Wine\LicenseInformation]
+		"msmpeg2adec-AACDecoderV2AddInEnable"=dword:00000001
+		"msmpeg2adec-AACDecoderV2InSKU"=dword:00000001
+		"msmpeg2adec-DolbyDigitalDecoderV2AddInEnable"=dword:00000001
+		"msmpeg2adec-DolbyDigitalDecoderV2InSKU"=dword:00000001
+		"msmpeg2vdec-H264VideoDecoderV2AddInEnable"=dword:00000001
+		"msmpeg2vdec-H264VideoDecoderV2InSKU"=dword:00000001
+		"msmpeg2vdec-MPEG2VideoDecoderV2AddInEnable"=dword:00000001
+		"msmpeg2vdec-MPEG2VideoDecoderV2InSKU"=dword:00000001
+
+		[HKEY_CLASSES_ROOT\CLSID\{271C3902-6095-4c45-A22F-20091816EE9E}]
+		@="MPEG4 Byte Stream Handler"
+
+		[HKEY_CLASSES_ROOT\CLSID\{271C3902-6095-4c45-A22F-20091816EE9E}\InprocServer32]
+		@="mf.dll"
+		"ThreadingModel"="Both"
+
+		[HKEY_CLASSES_ROOT\CLSID\{477EC299-1421-4bdd-971F-7CCB933F21AD}]
+		@="File Scheme Handler"
+
+		[HKEY_CLASSES_ROOT\CLSID\{477EC299-1421-4bdd-971F-7CCB933F21AD}\InprocServer32]
+		@="mf.dll"
+		"ThreadingModel"="Both"
+
+		[HKEY_CLASSES_ROOT\CLSID\{48e2ed0f-98c2-4a37-bed5-166312ddd83f}]
+		@="MFReadWrite Class Factory"
+
+		[HKEY_CLASSES_ROOT\CLSID\{48e2ed0f-98c2-4a37-bed5-166312ddd83f}\InprocServer32]
+		@="mfreadwrite.dll"
+		"ThreadingModel"="Both"
+		_EOF_
```

Comment:
This does not work because `_EOF_` is indented, so the shell doesn't terminate the here-document on this line
```suggestion
_EOF_
```